src/index.ts

```diff
@@ -41,19 +41,22 @@ async function getRealNames(repoList: ReturnType<typeof getAllRepoDeps>, config:
 				.then(content => {
 					const name = content?.name || ""
 					const version = content?.version || ""
-					return { name: name,
-						 version: version
-						}
+					const languageVersion = content?._nodeVersion || undefined
```

Comment:
why `_nodeVersion`, I though this would be `content.engines.node`?

Reply:
It's untyped for me so I didn't realise there was another way, it's changed now.